src/Browsershot.php

```diff
@@ -556,6 +556,15 @@ public function evaluate(string $pageFunction): string
         return $this->callBrowser($command);
     }
 
+    public function triggeredRequests(): array
+    {
+        $command = $this->createTriggeredRequestsListCommand();
+
+        $list = @json_decode($this->callBrowser($command), true);
```

Comment:
Do we need that `@` there?

Reply:
Used it to suppress any non-JSON formatted output, but now I think that throwing an exception gives more information about what happens rather than showing an empty array.